pkg/util/util.go

```diff
@@ -79,10 +79,11 @@ func GetLocalCluster(ss types.SubmarinerSpecification) (types.SubmarinerCluster,
 	localCluster.Spec.ColorCodes = ss.ColorCodes
 	return localCluster, nil
 }
+
```

Comment:
nit: unrelated change

Reply:
trying to improve readability - I like functions separated by new lines. I like what someone told me way back - "whitespace is your friend" :)